src/pint/models/timing_model.py

```diff
@@ -1706,17 +1706,23 @@ def compare(self, othermodel, nodmx=True, threshold_sigma=3.0, verbosity="max"):
                             newstr += " {:28f}".format(otherpar.value)
                         if otherpar.value != par.value:
                             sys.stdout.flush()
-                            log.warning(
-                                "Parameter %s not fit, but has changed between these models"
-                                % par.name
-                            )
+                            if par.name == "START" or par.name == "FINISH":
```

Comment:
Are these the only parameters that change without being fit? Fitter objects now set a number of parameters based on the results of the fit (e.g. EPHEM, NTOA, CHI2)

Reply:
No, not necessarily. Those parameters are just likely to change AND be relatively uninteresting between the two par files, while things like EPHEM, NTOA, and CHI2 might be good to highlight with the warning statements as opposed to info statements.

That said, it is quite easy to add more parameters to this exclude list, so if there are specific parameters that should be similarly "demoted," I can do that.